beads_db/src/main/resources/env/test_db.properties

```diff
@@ -0,0 +1,5 @@
+driver: com.mysql.jdbc.Driver
+url: jdbc:mysql://localhost:3306/beads3_test
```

Comment:
May be `beads_test` ?
and this DB can be completely empty because it should be used only by integration test and each test insert data that it needed. After test complete/fail transaction rollback and no data in DB